<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <machine name="local_alt" address="localhost" default="true"/>
    <node name="biotac_driver" pkg="biotac_sp_ros" type="biotac_sp_ros_single" output="screen"/>
    <node name="biotac_visualiser" pkg="biotac_sp_ros" type="visualiseBiotac_single.py" output="screen"/>
</launch>